ebtables/ebtables.go

```diff
@@ -51,6 +52,22 @@ func SetArpReply(ipAddress net.IP, macAddress net.HardwareAddr, action string) e
 	return executeShellCommand(command)
 }
 
+// SetEBRule sets an EB rule to route the given IP via the host.
+func SetEBRule(ipAddress string) error {
```

Comment:
rename this function to SetBrouteAccept and it should take 2 parmeters as like other function, 1 is ipaddress and other is action -> Either add or delete